include/swift/Basic/LLVM.h

```diff
@@ -89,4 +89,16 @@ namespace swift {
   using llvm::raw_ostream;
 } // end namespace swift
 
+/// Declares a const method with the given name and arguments intended for
+/// dumping. Methods declared with this macro should never be used except in the
+/// debugger; they are deprecated and marked used.
+#define DECLARE_DUMP_NAMED(name, ...) \
```

Comment:
This apparently doesn't work with empty parameter lists in the host compilers on Linux bots, and I'm not sure MSVC will take it either. I'll probably change it to something with a use site like (ignoring renaming):

```
DECLARE_DUMP_NAMED(dumpContext());
```

Reply:
Ooh, `void` will work.